arenadata/Dockerfile.ubuntu

```diff
@@ -42,14 +43,21 @@ ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
 ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
 
 # Compile with running mocking tests
-RUN bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash
+RUN set -eux; \
+    export PYTHON=/usr/bin/python2; \
+    bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash; \
+    cd gpdb_src/src/pl/plpython; \
+    make clean; \
+    export PYTHON=/opt/adb6-python3.9/bin/python3; \
+    cd /home/gpadmin; \
```

Comment:
may be
```suggestion
    pushd gpdb_src/src/pl/plpython; \
    make clean; \
    popd; \
    export PYTHON=/opt/adb6-python3.9/bin/python3; \
```

Reply:
pushd and popd are bash internal commands, not working in dockerfile by default

Reply:
you can always install
```Dockerfile
SHELL bash
```

Reply:
let's keep it simpler

Reply:
Do `make -C <PATH> <CMD>` is better that do `cd <PATH>`, `make <CMD>` and `cd`.

Reply:
Ok